### DISCUSSION TITLE
MacOS Camera issues

### DISCUSSION THREAD

**Discussion by Gizmo19699** [rgriebl/brickstore#720] [Q&A]
Hi,

Anyone else having issues with the camera working for Brickgonize within Brickstore?
My Webcam and FaceTime camera both work on my MacBook, but within the app i only get a black screen. I am on the latest update 4.4.   The camera was working on the previous version.

Any help would be appreciated!
Cheers!
Mark

**Comment by sophiakaile** [comment]
Check if the camera permission is enabled for Brickgonize within Brickstore. 
Restart your MacBook and try opening Brickstore again.
Try using a different camera application to see if the camera works in other apps.

**Reply by Gizmo19699** [reply]
Hi!

I have done all of the above.  Camera works with other apps, just not Brickstore anymore  :(
